docs/operation/cluster_monitoring/prometheus.rst

```diff
@@ -28,3 +28,20 @@ To customize predefined alert rules, refer to
 .. csv-table:: Default Prometheus Alerting rules
    :file: ../../../tools/rule_extractor/alerting_rules.csv
    :header: "Name", "Severity", "Description"
+
+Snapshot Prometheus Database
+----------------------------
+
+Snapshotting Prometheus database can be useful for backup and restore,
```

Comment:
If we mention backup/restore, we need to explain _how to restore_, which is not straightforward. I'd remove this part for now.

Reply:
I can remove it if you find it confusing.

Reply:
> a backup/restore of the Prometheus database itself

Yes I got that, my point was that restoring from a snapshot is not obvious, so if we mention this, we need to at least give pointers for how to do it (it's not even part of the official Prometheus docs). But anyway, that's also true about "later investigations", a user would need to know how to _use_ the generated snapshots. Maybe in a later iteration of this ticket.

Reply:
I've removed this paragraph.